src/lib/modules/ESSelector/ESSelectorModal.js

```diff
@@ -60,6 +62,9 @@ export default class ESSelectorModal extends Component {
             onSelectionsUpdate={this.onSelectionsUpdate}
             {...this.props}
           />
+          {warningMessage ? (
```

Comment:
I think the message does not belong with a selector component. It should be above it maybe?  In this way you add complexity (in already complicated place ;)) to the reusable component, just to be used this one time.